example/notebooks/pneumonia.ipynb

```diff
@@ -0,0 +1,1083 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Installing most needed libaries "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install scikit-learn foxai tqdm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "e_eezmSZJJe7"
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "import torch\n",
+    "import torchvision\n",
+    "import torch.nn as nn\n",
+    "import torch.optim as optim\n",
+    "from torchvision import datasets, models, transforms\n",
+    "from torch.utils.data.sampler import SubsetRandomSampler\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import time\n",
+    "import copy\n",
+    "from random import shuffle\n",
+    "\n",
+    "import tqdm.notebook as tqdm\n",
+    "\n",
+    "import sklearn\n",
+    "from sklearn.metrics import accuracy_score, cohen_kappa_score\n",
+    "from sklearn.metrics import classification_report\n",
+    "from PIL import Image\n",
+    "import cv2\n",
+    "\n",
+    "import os\n",
+    "import shutil"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Train Dataset Exploration and Creation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the purpose of this notebook we create dataset from 3 sources: \n",
+    "1. [Open dataset of Covid-19 cases with chest X-ray](https://github.com/ieee8023/covid-chestxray-dataset)\n",
+    "2. [Paul Mooney's pneumonia dataset from ](https://www.kaggle.com/datasets/paultimothymooney/chest-xray-pneumonia)\n",
+    "3. [Kaggle's Covid-19 radiography database](https://www.kaggle.com/datasets/tawsifurrahman/covid19-radiography-database)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_kg_hide-input": true,
+    "_kg_hide-output": true,
+    "id": "ZghMYdMHMGuj"
+   },
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv('./covid-chestxray-dataset/metadata.csv')\n",
+    "selected_df = df[df.finding==\"Pneumonia/Viral/COVID-19\"]\n",
+    "selected_df = selected_df[(selected_df.view == \"AP\") | (selected_df.view == \"PA\")]\n",
+    "selected_df.head(2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_kg_hide-input": true,
+    "id": "RADFgzxtKwbj"
+   },
+   "outputs": [],
+   "source": [
+    "images = selected_df.filename.values.tolist()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "yvdZqbdhLQLy"
+   },
+   "outputs": [],
+   "source": [
+    "shutil.rmtree('./COVID19-DATASET/train/covid19')\n",
```

Comment:
If there is no such file or directory it raises `FileNotFoundError`. You should check if it exists first.

Reply:
fixed